File: src/main/java/project/rpg/player/job/thing/Jobless.java
package project.rpg.player.job.thing;

import org.bukkit.entity.Player;
import project.rpg.player.job.JobBase;
import project.rpg.player.job.JobType;

public class Jobless extends JobBase {

    @Override
    public void reload() {
        //응 없어
    }

    @Override
    public void setNextJobs() {
        this._nextJobs.add(JobType.WARRIOR);
        this._nextJobs.add(JobType.WIZARD);
    }

    @Override
    public void setJobSkills() {
        //응 직업 스킬 없어
    }

    public Jobless(Player player) {
        super(JobType.JOBLESS, player);
    }

}

File: src/main/kotlin/project/rpg/player/job/thing/Jobless.kt
package project.rpg.player.job.thing

import org.bukkit.entity.Player
import project.rpg.player.job.JobBase
import project.rpg.player.job.JobType

class Jobless(player: Player) : JobBase(player, JobType.JOBLESS) {
    override fun reload() {
        //응 없어
    }

    override fun setNextJobs() {
        nextJobs.add(JobType.WARRIOR)
        nextJobs.add(JobType.WIZARD)
    }

    override fun setJobSkills() {
        //응 직업 스킬 없어
    }
}